iphone/package.json

```diff
@@ -18,8 +18,8 @@
 	"minIosVersion": "8.0",
 	"minWatchosVersion": "1.0",
 	"vendorDependencies": {
-		"xcode": ">=6.0 <=9.x",
```

Comment:
@sgtcoolguy Not bumping Xcode here, but we should require Xcode 8+ in Ti SDK 8.0.0, since we actually do via code already.

Reply:
These ranges are used by `ti info` to detect issues. I'd be fine with bumping this to `>=8.0 <=10.x`.